### PR TITLE
ID Localization for Resource Bin Packing.

### PR DESCRIPTION
**Pull request by ariscahyadi**
This PR addressing #20111

### CODE REVIEW

File: content/id/docs/concepts/configuration/resource-bin-packing.md [hunk 1]
```diff
@@ -0,0 +1,214 @@
+---
+title: _Bin Packing_ Sumber Daya untuk Sumber Daya Tambahan
```
Comment (sftim) [inline content/id/docs/concepts/configuration/resource-bin-packing.md:2]: Underscores in titles don't work
```suggestion
title: Bin Packing Sumber Daya untuk Sumber Daya Tambahan
```

File: content/id/docs/concepts/configuration/resource-bin-packing.md [hunk 2]
```diff
@@ -0,0 +1,214 @@
...
+---
+
+{{% capture overview %}}
+
+{{< feature-state for_k8s_version="1.16" state="alpha" >}}
+
+_Kube-scheduler_ dapat dikonfigurasikan untuk mengaktifkan _bin packing_ sumber 
+daya bersama dengan sumber daya tambahan melalui fungsi prioritas 
+`RequestedToCapacityRatioResourceAllocation`. Fungsi-fungsi prioritas dapat digunakan 
+untuk menyempurnakan _kube-scheduler_ sesuai dengan kebutuhan.
```
Comment (sftim) [inline content/id/docs/concepts/configuration/resource-bin-packing.md:14]: Is it worth explaining what “bin packing” means?

File: content/id/docs/concepts/configuration/resource-bin-packing.md [hunk 3]
```diff
@@ -0,0 +1,214 @@
...
+
+### Bagaimana Fungsi Prioritas RequestedToCapacityRatioResourceAllocation Menilai Node
+
+Bagian ini ditujukan bagi kamu yang ingin memahami secara detail internal
+dari fitur ini.
+Di bawah ini adalah contoh bagaimana nilai dari Node dihitung untuk satu kumpulan
+nilai yang diberikan.
+
+```
+Requested Resources
```
Comment (sftim) [inline content/id/docs/concepts/configuration/resource-bin-packing.md:122]: You could translate / localize this.

File: content/id/docs/concepts/configuration/resource-bin-packing.md [hunk 4]
```diff
@@ -0,0 +1,214 @@
...
+
+intel.com/foo : 2
+Memory: 256MB
+CPU: 2
+
+Resource Weights
```
Comment (sftim) [inline content/id/docs/concepts/configuration/resource-bin-packing.md:128]: You could translate / localize this.

File: content/id/docs/concepts/configuration/resource-bin-packing.md [hunk 5]
```diff
@@ -0,0 +1,214 @@
...
+
+intel.com/foo : 5
+Memory: 1
+CPU: 3
+
+FunctionShapePoint {{0, 0}, {100, 10}}
+
+Node 1 Spec
```
Comment (sftim) [inline content/id/docs/concepts/configuration/resource-bin-packing.md:136]: You could translate / localize “Spec”.

File: content/id/docs/concepts/configuration/resource-bin-packing.md [hunk 6]
```diff
@@ -0,0 +1,214 @@
...
+
+Available:
+intel.com/foo : 4
+Memory : 1 GB
+CPU: 8
+
+Used:
+intel.com/foo: 1
+Memory: 256MB
+CPU: 1
```
Comment (sftim) [inline content/id/docs/concepts/configuration/resource-bin-packing.md:146]: You could translate / localize “Available” and “Used”.

File: content/id/docs/concepts/configuration/resource-bin-packing.md [hunk 7]
```diff
@@ -0,0 +1,214 @@
...
+               = (100 -((8-3)*100/8))
+               = 37.5
+               = rawScoringFunction(37.5)
+               = 3
+
+NodeScore   =  (7 * 5) + (5 * 1) + (3 * 3) / (5 + 1 + 3)
+            =  5
+
+
+Node 2 Spec
```
Comment (sftim) [inline content/id/docs/concepts/configuration/resource-bin-packing.md:174]: You could translate / localize “Spec”.

File: content/id/docs/concepts/configuration/resource-bin-packing.md [hunk 8]
```diff
@@ -0,0 +1,214 @@
...
+Available:
+intel.com/foo: 8
+Memory: 1GB
+CPU: 8
+
+Used:
+
+intel.com/foo: 2
+Memory: 512MB
+CPU: 6
```
Comment (sftim) [inline content/id/docs/concepts/configuration/resource-bin-packing.md:185]: You could translate / localize “Available” and “Used”.

File: content/id/docs/concepts/configuration/resource-bin-packing.md (hunk 2 above)
Comment (ariscahyadi) [reply content/id/docs/concepts/configuration/resource-bin-packing.md:14]: Yes... but it may not totally correct.

**Comment by girikuncoro** [comment]
Great localization, thanks for your contribution with this page @ariscahyadi ! 🎉 

/lgtm
/approve

**Pull request merged**